### PR TITLE
ID-5881: fix flaky test sn 017 filter container

### PR DESCRIPTION
**Pull request by nuria-balaguer**

### CODE REVIEW

File: src/componentes/Filter/Filter.js [hunk 1]
```diff
@@ -128,12 +131,16 @@ const Filter = (props) => {
...
+            data-testid={`filter-${filterField}`}
             className={className}
             onOpen={load}
             active={filter.length > 0}
             button={button}
         >
-            <div className={classes.filterListContainer}>
+            <div
+                className={classes.filterListContainer}
+                data-testid={`filter-container-${filterField}`}
```
Comment (dimaPortyanka) [inline src/componentes/Filter/Filter.js:142]: Why do you need it twice?

File: src/componentes/Filter/FilterItem/FilterItem.js [hunk 2]
```diff
@@ -35,6 +35,7 @@ const FilterItem = (props) => {
             disabled={disabled}
             className={classes.root}
             onClick={onClick}
+            data-testid={`listItem-${label}`}
```
Comment (dimaPortyanka) [inline src/componentes/Filter/FilterItem/FilterItem.js:38]: FilterItem

File: src/componentes/Filter/Filter.js (hunk 1 above)
Comment (dimaPortyanka) [reply src/componentes/Filter/Filter.js:142]: if you need additional selectors classNames would fit better

Comment (nuria-balaguer) [reply src/componentes/Filter/Filter.js]: fixed

File: src/componentes/Filter/FilterItem/FilterItem.js (hunk 2 above)
Comment (nuria-balaguer) [reply src/componentes/Filter/FilterItem/FilterItem.js:38]: no, it has to be the label, as the filterItem is the main name (like locationName), and here I need the specific name (like DHL Global Fowarding)

**Review by dimaPortyanka — APPROVED** [review]

**Review by ricardovoegeli — APPROVED** [review]

**Pull request merged**